commands/shell.cmd

```diff
@@ -7,7 +7,9 @@ loadEnvConfig "${WARDEN_ENV_PATH}" || exit $?
 ## set defaults for this command which can be overridden either using exports in the user
 ## profile or setting them in the .env configuration on a per-project basis
 WARDEN_ENV_SHELL_COMMAND=${WARDEN_ENV_SHELL_COMMAND:-bash}
-WARDEN_ENV_SHELL_CONTAINER=${WARDEN_ENV_SHELL_CONTAINER:-php-fpm}
+
+WARDEN_ENV_DEFAULT_SHELL_CONTAINER=$([ $WARDEN_PHP_SPX == 1 ] && echo "php-spx" || echo "php-fpm")
```

Comment:
@SamJUK I don't fully agree with this. The `warden shell` goes to the standard php-fpm container and `warden debug` goes to php-debug container. To me this would seem like we could either add a flag `--spx` or create a new `warden spx` command to drop to a shell in it.

Reply:
Makes sense, I'll switch it over to a separate command `warden spx` (since AFAIK the flag approach is not widely used across the project)